openapi/components/schemas/CreditMemos/CreditMemoProperties.yaml

```
@@ -0,0 +1,176 @@
+id:
+  description: Credit memo ID.
+  readOnly: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+invoiceId:
+  description: ID of the invoice which the credit memo is issued upon.
+  nullable: true
+  allOf:
+    - $ref: ../ResourceId.yaml
+customerId:
+  $ref: ../CustomerId.yaml
+number:
+  description: An auto-incrementing number based on the sequence of credit memos for any particular customer.
+  readOnly: true
+  type: integer
+allocations:
+  type: array
+  items:
+    type: object
+    properties:
+      transactions:
+        type: array
+        items:
+          type: object
+          properties:
+            transactionId:
+              description: ID of the transaction that the credit memo is allocated to.
```

nitpick: suggestion to avoid ending in a preposition
```suggestion
              description: ID of the transaction on which the credit memo is allocated.
```